client/components/List/List.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropsWithChildren } from 'react';
+
+interface Props {
+  variant?:
+  | 'none'
+  | 'with--bullets'
+  | 'with--checkmark'
+  | 'with--count',
+  type: 'ul' | 'ol',
+  className?: string,
+}
+
+function List({
+  variant = 'none', children, type, className = '',
+} : PropsWithChildren<Props>) {
+  return (
+    type === 'ul' ? (
```

Comment:
We can simplify this code and have something similar to the `FormattedText` component. Look there for inspiration.

PS: I think instead of `Wrapper` here we should name it `List` so it's clearer what is going on.